tests/unit/task_scheduler/file/test_backend.py

```
@@ -64,6 +64,29 @@ def test_execute_function_mixed(self):
+        self.assertTrue(future_file_obj.done())
+        self.assertEqual(future_file_obj.result(), 3)
+
+    def test_backend_write_file_serialization_error(self):
+        cache_directory = os.path.abspath("executorlib_cache")
+        os.makedirs(cache_directory, exist_ok=True)
+        file_name = os.path.join(cache_directory, "test_file_i.h5")
+        backend_write_file(file_name=file_name, output={"result": Future()}, runtime=0.1)
+        future_file_obj = FutureItem(
+            file_name=os.path.join(cache_directory, "test_file_o.h5")
+        )
+        with self.assertRaises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)